tests/test_logging.js

```diff
@@ -22,28 +24,26 @@ test('match debugging works', async t => {
   const exampleBody = 'Hello yourself!'
   await got.post('http://example.test/deep/link', { body: exampleBody })
 
-  t.ok(
-    log.calledWith(
-      sinon.match.string,
-      sinon.match('http://example.test/deep/link'),
-      sinon.match(exampleBody)
-    )
+  expect(logFn).to.have.been.calledWithExactly(
+    sinon.match.string,
+    sinon.match('http://example.test/deep/link'),
+    sinon.match(exampleBody)
```

Comment:
Are the `match` calls needed here? I thought you could just pass strings in directly.

Reply:
Yea, they are; I reworked them a bit to clarify why.